src/status_im/translations/en.cljs

```diff
@@ -39,17 +39,17 @@
    :status-console                       "Status Console"
 
    ;;sign in
-   :intro-text                           "Status is an open source decentralized chat and Ethereum browser"
-   :intro-text-description               "Status is built with the help of the community to help you use all the benefits of decentralized web in your mobile phone"
+   :intro-text                           "Status is your gateway to the decentralized web"
+   :intro-text-description               "A community-powered platform to securely chat and transact on the Ethereum blockchain"
    :create-account                       "Create account"
    :already-have-account                 "I already have an account"
-   :creating-your-account                "Creating your account on the blockchain. We can't touch it, no one can, except for you!"
-   :password-placeholder                 "Type your password"
+   :creating-your-account                "Creating your account on the blockchain..."
```

Comment:
@obi2020 Weren't you going to update the `on the blockchain` part as per this comment?

https://github.com/status-im/status-react/issues/5088#issuecomment-403824589

So for example, "Generating your account" might be more accurate?

Reply:
Ah, mistake. "on the blockchain" was meant to be erased too